src/main/java/edu/umich/tl/ApiCallHandler.java

```diff
@@ -101,7 +173,10 @@ private HttpResponse apiDirectCanvas(String url) {
 				clientRequest.addHeader(h.getName(), h.getValue());
 			}
 			try {
+				Stopwatch stopwatch = Stopwatch.createStarted();
 				httpResponse = client.execute(clientRequest);
+				stopwatch.stop();
```

Comment:
The stopwatch.stop and log message should be in a finally block so that they are always printed.  Currently they won't be printed if there is an exception.